trackio/__init__.py

```diff
@@ -284,7 +285,15 @@ def show(
             functions will be added as MCP tools. If `None` (default behavior), then the
             `GRADIO_MCP_SERVER` environment variable will be used to determine if the
             MCP server should be enabled (which is `"True"` on Hugging Face Spaces).
+        color_palette (`list[str]`, *optional*):
```

Comment:
Idea for future: support named palette, like "pastel", "rainbow" etc

Reply:
Yep sounds good!